### ISSUE TITLE
GitHub Action's deprecation warnings : set-output

### ISSUE THREAD

**Issue by laurentvasseurtekos** [kanga333/variable-mapper#102]
Hello, thank you for this Github action is really useful. 

I would like to notice that since this github action update : [GitHub Actions: Deprecating save-state and set-output commands](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) , the action seems to generate this warning : 

```
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```
![image](https://user-images.githubusercontent.com/96220327/196601784-f2a3862c-96c3-48a6-8aa0-57f5e132c33c.png)

Have a good day

**Comment by kanga333** [comment]
Thank you for reporting.
I've updated packages and resolved this issue by #103.
Could you try `kanga333/variable-mapper@v0.3.0` ?

**Comment by laurentvasseurtekos** [comment]
I confirm it is good ! Thank you a lot !
